src/reactionsystem.jl

```diff
@@ -1469,6 +1471,8 @@ function spatial_convert_err(rs::ReactionSystem, systype)
     isspatial(rs) && error("Conversion to $systype is not supported for spatial networks.")
 end
 
+COMPLETENESS_ERROR = "A ReactionSystem must be complete before it can be converted to other system types. A ReactionSystem can be marked as complete using the `compelte` function."
+
```

Comment:
Long text that is used repeatedly. When we refactor this file we might want to put it in a better place.